txstratum/protocol.py

```diff
@@ -372,8 +372,22 @@ def method_submit(self, params: Any, msgid: JSONRPCId) -> None:
             # to solve the block.
             self.manager.submit_solution(self, job, obj.get_struct_nonce())
             if obj.is_block:
+                self.log.info(
+                    "Block solved: ",
+                    miner_id=self.miner_id,
```

Comment:
I guess you could add the block hash so you can filter out duplicated log entries.

Reply:
Done https://github.com/HathorNetwork/tx-mining-service/pull/115/commits/6a03c3952974bf811598bdc1cbfcdc149ed9584d